pkg/oc/cli/admin/migrate/scc/scc.go

```diff
@@ -278,3 +303,46 @@ func convertSccToClusterRoleBinding(scc *securityapi.SecurityContextConstraints)
 
 	return &binding, nil
 }
+
+func convertClusterRolesWithSCCRules(scc *securityapi.SecurityContextConstraints, roles *rbacv1.ClusterRoleList) (rolesToUpdate []rbacv1.ClusterRole) {
+	for roleIt := 0; roleIt < len(roles.Items); roleIt++ {
+		if convertRulesWithSCCName(&roles.Items[roleIt].Rules, scc.Name) {
+			rolesToUpdate = append(rolesToUpdate, roles.Items[roleIt])
+		}
+	} // ClusterRoles
+	return
+}
+
+func convertRolesWithSCCRules(scc *securityapi.SecurityContextConstraints, roles *rbacv1.RoleList) (rolesToUpdate []rbacv1.Role) {
+	for roleIt := 0; roleIt < len(roles.Items); roleIt++ {
+		if convertRulesWithSCCName(&roles.Items[roleIt].Rules, scc.Name) {
+			rolesToUpdate = append(rolesToUpdate, roles.Items[roleIt])
+		}
+	} // Roles
+	return
+}
+
+func convertRulesWithSCCName(rules *[]rbacv1.PolicyRule, sccName string) (changed bool) {
```

Comment:
Consider adding a comment, so I won't need to read all these loops and ifs to understand what's going on.

Reply:
Will do.